devices/bosch/room_thermostat2_230V.json

```diff
@@ -0,0 +1,451 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": "BOSCH",
+  "modelid": "RBSH-RTH0-ZB-EU",
+  "vendor": "Bosch",
+  "product": "Room thermostat II 230V (BTH-RM230Z)",
+  "sleeper": false,
+  "status": "Gold",
+  "subdevices": [
+    {
+      "type": "$TYPE_THERMOSTAT",
+      "restapi": "/sensors",
+      "uuid": [
+        "$address.ext",
+        "0x01",
+        "0x0201"
+      ],
+      "fingerprint": {
+        "profile": "0x0104",
+        "device": "0x0301",
+        "endpoint": "0x01",
+        "in": [
+          "0x0001",
```

Comment:
Please remove entry `0x0001` here, we do not have a battery powered device this time 😉